constants/chatbotCommands.py

```diff
@@ -681,40 +683,32 @@ async def getPPMessage(userID: int, just_data: bool = False) -> Any:
     currentMap = current_info["beatmap_id"]
     currentMods = current_info["mods"]
     currentAcc = current_info["accuracy"]
+    currentMode = 0
+    currentMisscount = 0
+
+    beatmap = await beatmap_utils.id_from_api(currentMap, should_save=False)
+    if not beatmap:
+        return "Could not retrieve beatmap information for the given map."
+
+    est_performance, est_star_rating = await performance_utils.calculate_performance(
+        currentMap,
+        currentMode,
+        currentMods,
+        beatmap.max_combo,
+        currentAcc,
+        currentMisscount,
+    )
 
-    # Send request to score-service to calculate pp
-    try:
-        response = await glob.http_client.get(
-            url=f"{settings.SCORE_SERVICE_BASE_URL}/api/v1/pp",
-            params={"b": currentMap, "m": currentMods},
-            timeout=5,
-        )
-    except:
-        logger.exception(
-            "Failed to retrieve PP from LESS API",
-            extra={
-                "user_id": userID,
-                "beatmap_id": currentMap,
-                "mods": currentMods,
-            },
-        )
-        return "Score server currently down, could not retrieve PP."
-
-    if not response or response.status_code != 200:
-        return "API Timeout. Please try again in a few seconds."
-
-    data = response.json()
-
-    # Make sure status is in response data
-    if "status" not in data:
-        return "Unknown error in LESS API call."
-
-    # Make sure status is 200
-    if data["status"] != 200:
-        if "message" in data:
-            return f"Error in LESS API call ({data['message']})."
-        else:
-            return "Unknown error in LESS API call."
+    data = {
+        "status": 200,
+        "message": "ok",
+        "song_name": beatmap.song_name,
+        "pp": est_performance,
+        "length": beatmap.hit_length,
+        "stars": est_star_rating,  # TODO is this wrong for common values?
+        "ar": beatmap.ar,
+        "bpm": beatmap.bpm,
+    }
```

Comment:
matching the api that was being called : https://github.com/osuAkatsuki/score-service/blob/6a8c37688a10e20dbdb5ab022cf7201edff6f3b2/app/api/pp.py#L79-L88

not really trying to deeply refactor chatbotCommands rn